corehq/apps/es/users.py

```diff
@@ -30,7 +30,7 @@
 class UserES(HQESQuery):
     index = 'users'
     default_filters = {
-        'not_deleted': {"term": {"base_doc": "couchuser"}},
+        'not_deleted': {"term": {"base_doc": "CouchUser"}},
```

Comment:
Great catch, I could have sworn I had this working locally but when I just tested it you are correct.  Reverted in 1631f7d